lib/mixins/dropdown.js

```diff
@@ -181,11 +182,10 @@ export default {
             });
         },
         focusItem(idx, items) {
-            items.forEach((el, i) => {
-                if (i === idx) {
-                    el.focus();
-                }
-            });
+            let el = items.find((el, i) => i === idx)
```

Comment:
There are already `.find`s in the lib: 
![image](https://user-images.githubusercontent.com/5158436/29218785-217fb968-7ecb-11e7-88f5-de071c889f4b.png)

I think we can easily use babel to transpile `.find` into legacy implementation :)